src/tiled/commanddialog.cpp

```diff
@@ -81,7 +84,7 @@ void CommandDialog::updateKeySequenceEdit(const QModelIndex &current, const QMod
     if (current.row() < mUi->treeView->model()->rowCount(QModelIndex()) - 1)
         mUi->keySequenceEdit->setKeySequence(mUi->treeView->model()->shortcut(current));
     else
-        mUi->keySequenceEdit->clear();
```

Comment:
It should still be cleared. It makes no sense to show a disabled widget with a shortcut from the previously selected command.